component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```
@@ -242,7 +245,27 @@ public Response getUserAvatarById(@Context UriInfo uriInfo,
     builder.cacheControl(cc);
     return builder.cacheControl(cc).build();
   }
-  
+
+  private Response getDefaultUserAvatar(String avatarUrl) {
+    try {
+      if (avatarUrl != null) {
+        URL url = new URL(avatarUrl);
+        String type = url.openConnection().getHeaderField("Content-Type");
+        if (type != null && type.startsWith("image/")) {
+          InputStream input = url.openStream();
+          return Response.ok(input, type).build();
+        }
+      }
+    } catch (IOException ignored) {
```

Avoid empty catch block. We should at least add a log.debug IMO.